docs/intro/pages/Navigation.html

```diff
@@ -0,0 +1,331 @@
+                                Colors
+                            </a>
+                        </li>
+                        <li>
+                            <a  class="text-mute" href="intro/pages/typography.html">
+                                Typography
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/avatar.html">
+                                Avatar
+                            </a>
+                        </li>
+
+                        <li>
+                            <a  class="text-mute" href="intro/pages/alert.html">
+                                Alert
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/badge.html">
+                                Badge
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/button.html">
+                                Button
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/card.html">
+                                Card
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/images.html">
+                                Image
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/input.html">
+                                Input
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/list.html">
+                                list
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/modal.html">
+                                Modal
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/navigation.html">
+                                Navigation
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="text-mute" href="intro/pages/grid.html">
+                                Grid layout
+                            </a>
+                        </li>   
+
+                    </ul>
+                </div>
+                
+
+            </div>
+            <!--- left sidebar -->
+
+
+            <!--- Right container -->
+            <div class="right-container p-s">
+              
```

Comment:
Use prettier. Avoid giving extra spaces.

Reply:
I try to prettier but it is not working so I do myself